.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Tests
+
+on:
+  pull_request:
+    paths-ignore:
+    - 'doc/**'
+  push:
+    branches:
+    - main
+
+jobs:
+  Linux:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: bazelbuild/setup-bazelisk@v2
+      - name: install dependencies
+        run: |
+            export DEBIAN_FRONTEND=noninteractive
+            sudo apt-get install -y build-essential git clang libc++-dev
+      - name: bazel tests
+        run: |
+            bazel test --verbose_failures --test_output=errors //...
```

Comment:
We run `@capnp-cpp//... //... -- -@capnp-cpp//src/capnp:fuzz-test`, do we want capnp tests here too?